.github/workflows/tests.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         with:
           name: bazel-testlogs
           path: bazel-testlogs
+
```

Comment:
I noticed that we have newlines between half of the CI jobs but not others. This makes it conistent.